spec/services/scrapers/medium_priority_spec.rb

```diff
@@ -1,9 +1,18 @@
 require 'rails_helper'
+require 'sidekiq/testing'
+Sidekiq::Testing.fake!
 
 RSpec.describe Scrapers::MediumPriority do
   describe '#perform' do
-    it 'add specs' do
-      skip
+    it 'adds jobs to the CourtCaseWorker' do
+      test_case = create(:court_case, :with_html, closed_on: '6/15/2022')
+      test_case.case_html.update(scraped_at: '6/15/2022')
+      false_case = create(:court_case, :with_html, closed_on: '6/11/2022')
+      false_case.case_html.update(scraped_at: '7/11/2022')
```

Comment:
Similar to the [other PR comment](https://github.com/AyOK-Code/oscn/pull/77/files#r920453990), you'll need to take into account the date that the test is being run on. As time passes, this spec will start to fail. See other PR for solutions